modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/resource/v1_0/SXPBlueprintResourceImpl.java

```diff
@@ -94,15 +94,18 @@ public SXPBlueprint getSXPBlueprintByExternalReferenceCode(
 			String externalReferenceCode)
 		throws Exception {
 
+		com.liferay.search.experiences.model.SXPBlueprint sxpBlueprint =
```

Comment:
Is there a reason not to inline this, as it was?

Reply:
@peerkar My bad. I did this to change the parameter on the DefaultDTOConverterContext, I put externalReferenceCode, but its require an ID, I will fix this.